scenarios/ocd/CreateAKSDeployment/README.md

```diff
@@ -315,7 +279,7 @@ Run the following command to get the HTTPS endpoint for your application:
 >[!Note]
 > It often takes 2-3 minutes for the SSL certificate to propogate and the site to be reachable via https 
 ```bash
-echo https://$FQDN
+curl https://$FQDN
```

Comment:
Same as the previous curl command, should the timeout/retry be set so that the request doesn't fail before the SSL certificates become ready?